### DISCUSSION TITLE
用4.5版本做区域GLDAS驱动数据问题求助

### DISCUSSION THREAD

**Discussion by 12wlj** [NCAR/hrldas#227] [Q&A]
你好，我在用hrldas-v4.5_master/hrldas/HRLDAS_forcing做GLDAS驱动数据时候，设置好namelist.input.GLDAS，运行create_forcing.exe，发生如图所示的报错情况。我检查了GLDAS_Qair_f_inst_2021010100是存在Qair_f_inst变量的，是Can't read variable: Q_f_inst的问题吗，
![Uploading 5.png…]()

**Comment by cenlinhe** [comment]
did you follow up the instruction here: https://github.com/NCAR/hrldas/blob/7d183b9ba6ff82f746a978fb20bf79e1cb4f1466/hrldas/docs/README.GLDAS#L42-L176?

there are two additional discussions for references
https://github.com/NCAR/hrldas/discussions/91
https://github.com/NCAR/hrldas/discussions/103
